external/header.php

```diff
@@ -34,21 +54,29 @@ public static function __callstatic($name, $arguments)
 // Only users from authorized IP addresses may control Profiling
 if ($controlIPs === false || in_array($_SERVER['REMOTE_ADDR'], $controlIPs) || PHP_SAPI == 'cli')
 {
-  if (isset($_GET['_profile']))
+  /* Backwards Compatibility getparam check*/
+  if( !isset($_xhprof['getparam']) )
```

Comment:
I remember you said that this change will come in separate PR after this one is merged. I guess it was included in this PR by mistake.